app/js/controllers/sellCtrl.js

```diff
@@ -64,22 +64,15 @@
 
         function getCurrencies()
         {
-            CurrenciesService.getCurrencies()
-                    .then(function (data)
-                    {
-                        var rates = data;
-                        for (var key in rates) {
-                            if (rates[key].code === ctrl.currency) {
-                                ctrl.rate = rates[key];
-                            }
-                        }
-                        ctrl.sellCost = sellCost;
-
-                    })
-                    .catch(function (error)
-                    {
-                        console.log('Error ', error);
-                    });
+            ctrl.rates = RatesFactory.getRates();
+            angular.forEach(ctrl.rates, function (rate)
+            {
+                if (rate.code === ctrl.currency) {
+                    ctrl.rate = rate;
+                }
+            });
+            ctrl.sellCost = sellCost;
```

Comment:
SellCost is a function which calculate how many pln we will get when sell some other money